### PR TITLE
Update the cyclonedx dependency

### PR DESCRIPTION
**Pull request by uvifian**

### CODE REVIEW

**Comment by uvifian** [comment]
Jenkins: please test

**Comment by Wattos** [comment]
Urs, please wait with merging until we are done with log4j

**Comment by uvifian** [comment]
@Wattos sure, I have not even selected any reviewers... But I think I just found one :-)

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 org.gradle.caching=true
-version=0.9.23
+version=0.9.24
```
Comment (Wattos) [inline gradle.properties:2]:
```suggestion
version=0.9.25
```

This version is already being used

**Review by Wattos — APPROVED** [review]
Looks good!

**Pull request merged**